tests/components/dunehd/test_config_flow.py

```diff
@@ -64,6 +64,16 @@ async def test_user_invalid_host(hass):
     assert result["errors"] == {CONF_HOST: "invalid_host"}
 
 
+async def test_user_very_long_host(hass):
+    """Test that errors are shown when the host is longer than 253 chars."""
+    long_host = "very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host_very_long_host"
```

Comment:
It's nice to break long strings around 88 characters.

Reply:
OK, I'll change this.

Reply:
Done https://github.com/home-assistant/core/pull/49900